jetstream/cli.py

```diff
@@ -182,12 +182,11 @@ def export_statistics_to_json(project_id, dataset_id, bucket):
 @cli.command("rerun_config_changed")
 @project_id_option
 @dataset_id_option
-@dry_run_option
-def rerun_config_changed(project_id, dataset_id, dry_run):
+def rerun_config_changed(project_id, dataset_id):
```

Comment:
BigQuery access is required to get the last modified timestamps from the tables. Since we don't have access to the data during a dry run, I removed the `dry_run` option here.